tests/dualtor/test_orchagent_standby_tor_downstream.py

```diff
@@ -209,34 +209,6 @@ def test_standby_tor_downstream_loopback_route_readded(ptfhost, rand_selected_du
     check_tunnel_balance(**params)
 
 
-def build_packet_to_server(tor, ptfadapter, target_server_ip, tunnel_traffic_monitor):
```

Comment:
Is this method already defined somewhere else?

Reply:
A recent PR moved this to dual_tor_utils:

https://github.com/Azure/sonic-mgmt/blob/06a13d4b02148dde8c7acb8d1a55ed7f024e318b/tests/common/dualtor/dual_tor_utils.py#L803

Reply:
This PR: https://github.com/Azure/sonic-mgmt/pull/3349

Reply:
Ah yes, thanks!